### PR TITLE
refactor(DiskStateProgressBar): clarify severity references

### PR DESCRIPTION
**Pull request by ariser**
Disk severity references were overcomplicated and unclear, this simplifies them.

### CODE REVIEW

File: src/containers/Storage/Pdisk/Pdisk.tsx [hunk 1]
```diff
@@ -8,38 +8,37 @@ import {InternalLink} from '../../../components/InternalLink';
...
-    [TPDiskState.InitialFormatReadError]: 5,
-    [TPDiskState.InitialSysLogReadError]: 5,
-    [TPDiskState.InitialSysLogParseError]: 5,
-    [TPDiskState.InitialCommonLogReadError]: 5,
-    [TPDiskState.InitialCommonLogParseError]: 5,
-    [TPDiskState.CommonLoggerInitError]: 5,
-    [TPDiskState.OpenFileError]: 5,
-    [TPDiskState.ChunkQuotaError]: 5,
-    [TPDiskState.DeviceIoError]: 5,
+    [TPDiskState.Initial]: EDiskStateSeverity.Grey,
```
Comment (ariser) [inline src/containers/Storage/Pdisk/Pdisk.tsx:28]: those are direct references to the possible component prop values, should use enum

File: src/containers/Storage/Pdisk/Pdisk.tsx [hunk 2]
```diff
@@ -75,9 +74,9 @@ function Pdisk(props: PDiskProps) {
     const preparePdiskData = () => {
         const {AvailableSize, TotalSize, State, PDiskId, NodeId, Path, Realtime, Device} = props;
         const errorColors = [
-            diskProgressColors[colorSeverity.Orange as keyof typeof diskProgressColors],
-            diskProgressColors[colorSeverity.Red as keyof typeof diskProgressColors],
-            diskProgressColors[colorSeverity.Yellow as keyof typeof diskProgressColors],
+            EFlag.Orange,
+            EFlag.Red,
+            EFlag.Yellow,
```
Comment (ariser) [inline src/containers/Storage/Pdisk/Pdisk.tsx:79]: these were unnecessary complicated references to color names
the only purpose of this array is to check whether fields contain values that should be treated as 'errors'

There is room to improvement: seems like what considered an 'error' is based on the `DiskStateProgressBar` appearance, thus the component should be the source of truth for determining whether a color is an 'error' color. But this is another refactoring.

File: src/containers/Storage/Vdisk/Vdisk.js [hunk 3]
```diff
@@ -37,20 +36,20 @@ const propTypes = {
 };
 
 const stateSeverity = {
-    Initial: 3,
-    LocalRecoveryError: 5,
-    SyncGuidRecoveryError: 5,
-    SyncGuidRecovery: 3,
-    PDiskError: 5,
-    OK: 1,
+    Initial: EDiskStateSeverity.Yellow,
```
Comment (ariser) [inline src/containers/Storage/Vdisk/Vdisk.js:39]: same, those are direct references to the component severity

File: src/containers/Storage/Vdisk/Vdisk.js [hunk 4]
```diff
@@ -111,14 +110,14 @@ function Vdisk(props) {
         PoolName && vdiskData.push({label: 'StoragePool', value: PoolName});
 
         SatisfactionRank &&
-            SatisfactionRank.FreshRank?.Flag !== diskProgressColors[colorSeverity.Green] &&
+            SatisfactionRank.FreshRank?.Flag !== EFlag.Green &&
```
Comment (ariser) [inline src/containers/Storage/Vdisk/Vdisk.js:113]: same with overcomplicated references to colors

File: src/containers/Storage/utils/constants.ts [hunk 5]
```diff
@@ -1,10 +1,3 @@
-export const colorSeverity = {
```
Comment (ariser) [inline src/containers/Storage/utils/constants.ts:1]: this was actually a reverse map for `DiskStateProgressBar` severity, solved with introducing `enum EDiskStateSeverity`

**Review by artemmufazalov — APPROVED** [review]

**Pull request merged**